app/src/main/java/wordtextcounter/details/main/util/RateUsHelper.kt

```diff
@@ -30,12 +33,15 @@ object RateUsHelper {
       dialog.window.setBackgroundDrawable(ColorDrawable(Color.TRANSPARENT))
       val btClose = dialog.findViewById<ImageView>(R.id.ivCross)
       val btYes = dialog.findViewById<Button>(R.id.btnSave)
```

Comment:
Can use views directly from Kotlin Android Extensions as we have contentview. No need for findviewbyid